jobs/build/update-base-images/build.sh

```diff
@@ -35,7 +40,7 @@ build_common() {
     LABEL \\
             com.redhat.component=\"openshift-enterprise-base-container\" \\
             name=\"openshift/ose-base\" \\
-            version=\"v4.0\" \\
+            version=\"${TAG}\" \\
```

Comment:
i'm afraid you can't use `rhaos-4.1-rhel-8` as a version :) also we don't want to - please leave this hardcoded as `v4.0` (the fact that it comes from a different branch really doesn't matter, version is rather arbitrary, and `v4.0` can't be confused with any release-specific usage).